src/python/geoclaw/surge/storm.py

```diff
@@ -548,6 +549,115 @@ def read_ibtracs(self, path, storm_name, year):
             self.storm_radius = numpy.where(ds.usa_roci >=0,
                 units.convert(ds.usa_roci.values,'nmi','m'),-1)
 
+    def read_emanuel(self, path, storm_name, verbose=False):
+        r"""Read in Kerry Emanuel's storm file
+
+        This reads in the netcdf-formatted tracks from Kerry Emanuel.
+        Correspondence September 2018. Original storms have been converted
+        from matlab to netcdf format by mlimb@rhg.com and are at
+        /gcs/../coastal/storm_tracks/emmanuel_projections_2018-09-12/netcdf_added_velocity/.
+
+        :Input:
+        - *path* (string) Path to the file to be read.
+        - *storm_name* (string) storm name containing information about
+        model, scenario, time period, and storm index within the file.
+            ex. ccsm4_rcp85_2007_2025_0, hadgem5_rcp45_2035_2045_10
+            model_scenario_period is equivalent to filename.
+
+        Assumes these variables exist in Emanuel dataset:
+           datetime
+           longstore
+           latstore
+           rmstore (central max wind radius)
+           pstore (central pressure)
+           v_total_max_ms
+           bas (attribute)
+
+        :Raises:
+        - ImportError if xarray not found
+        - KeyError if provided storm name is not found in
+        given file (assumes path is correct)
+        """
+        # try/except copied from read_ibtracs
+        # imports that you don't need for other read functions
+        try:
+            import xarray as xr
+        except ImportError as e:
+            print("Emanuel tracks currently require xarray "
+                  "to work.")
+            raise e
+
+        def _convert_to_python_datetime(np_date):
+            '''Convert numpy.datetime64 to python datetime.datetime'''
+            (year, month, day_hour) = str(np_date.values).split('-')
+            day = day_hour[0:2]
+            hour = day_hour[3:5]
+            return datetime.datetime(int(year), int(month), int(day), int(hour))
+
+        storm_index = int(storm_name.split('_')[-1])
```

Comment:
just heads up: currently, I'm not checking if input model name is valid.

for example, if  file path for `ccsm4_rcp85_2007_2025.nc` is given, even if incorrect storm name is given  `hadgem5_rcp45_2035_2045_0` instead of `ccsm4_rcp85_2007_2025_0`, `read_emanuel` would still retrieve 0th storm. But `storm_index` assumes correct name format is given.

Reply:
> @bolliger32 just heads up: currently, I'm not checking if input model name is valid.
> 
> for example, if file path for `ccsm4_rcp85_2007_2025.nc` is given, even if incorrect storm name is given `hadgem5_rcp45_2035_2045_0` instead of `ccsm4_rcp85_2007_2025_0`, `read_emanuel` would still retrieve 0th storm. But `storm_index` assumes correct name format is given.

Sounds good thanks for the heads up!

Reply:
thanks for your work on this @moonlimb!